events/WAFV2/UpdateIPSet.json

```diff
@@ -9,6 +9,27 @@
     "mitreAttackTechniques": [
         "T1562 - Impair Defenses"
     ],
+    "mitreAttackSubTechniques": [
+
+    ],
+    "unverifiedMitreAttackTechniques": [
+        {
+            "technique": "T1071 - Application Layer Protocol",
+            "reason": "By updating an IPSet to allow or block specific IP addresses, an attacker can manipulate web traffic to facilitate or evade detection during Command and Control activities."
+        },
+        {
+            "technique": "T1078 - Valid Accounts",
+            "reason": "An attacker with legitimate credentials could update the IPSet to include malicious IPs, ensuring ongoing access or enabling external connections from trusted addresses."
```

Comment:
I agree but as mentioned in other techniques, the use of legitimate credentials is needed in all techniques.

Reply:
Removed.